apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -595,9 +595,7 @@ const BookingPage = ({
                       required
                       className={classNames(
                         inputClassName,
-                        bookingForm.formState.errors.email
-                          ? "border-red-700 focus:ring-red-700"
-                          : " border-gray-300  dark:border-gray-900"
+                        bookingForm.formState.errors.email && "!focus:ring-red-700 !border-red-700"
```

Comment:
This was causing issues so just adding important if there is errors (how it should be imho)